src/transformers/models/layoutlmv2/tokenization_layoutlmv2.py

```diff
@@ -0,0 +1,45 @@
+# coding=utf-8
+from transformers.models.layoutlm.tokenization_layoutlm import LayoutLMTokenizer
+from transformers.utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+VOCAB_FILES_NAMES = {"vocab_file": "vocab.txt"}
+
+PRETRAINED_VOCAB_FILES_MAP = {
+    "vocab_file": {
+        "microsoft/layoutlmv2-base-uncased": "https://huggingface.co/microsoft/layoutlmv2-base-uncased/resolve/main/vocab.txt",
+        "microsoft/layoutlmv2-large-uncased": "https://huggingface.co/microsoft/layoutlmv2-large-uncased/resolve/main/vocab.txt",
+    }
+}
+
+
+PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES = {
+    "microsoft/layoutlmv2-base-uncased": 512,
+    "microsoft/layoutlmv2-large-uncased": 512,
+}
+
+
+PRETRAINED_INIT_CONFIGURATION = {
+    "microsoft/layoutlmv2-base-uncased": {"do_lower_case": True},
+    "microsoft/layoutlmv2-large-uncased": {"do_lower_case": True},
+}
+
+
+class LayoutLMv2Tokenizer(LayoutLMTokenizer):
```

Comment:
I think it's better to let it inherit from `BertTokenizer` rather than `LayoutLMTokenizer` (since all tokenizers in HuggingFace Transformers that are identical to BERT's tokenizer inherit from `BertTokenizer`).